### PR TITLE
New content EN FAQ

### PR DESCRIPTION
**Pull request by typ1-brods**

### CODE REVIEW

File: docs/EN/Getting-Started/FAQ.md [hunk 1]
```diff
@@ -88,8 +88,20 @@ You can remove the pump while taking a shower or bath. For this short period of
 
 ## Sleeping
 
+### How can I loop during the night without mobile and WLAN radiation?
```
Comment (AdrianLxM) [inline docs/EN/Getting-Started/FAQ.md]: I believe the more common term in English is WIFI.

File: docs/EN/Getting-Started/FAQ.md [hunk 2]
```diff
@@ -88,8 +88,20 @@ You can remove the pump while taking a shower or bath. For this short period of
...
+Many users turn the phone into airplane mode at night. If you want the loop to support you when you are sleeping, proceed as follows (this will only work with a local BG-source such as xDrip + or patched Dexcom app, it will NOT work if you get the BG-readings via nightscout):
```
Comment (AdrianLxM) [inline docs/EN/Getting-Started/FAQ.md]: xDrip+ as one word

**Comment by typ1-brods** [comment]
Thank you Adrian. Fixed.

File: docs/EN/Getting-Started/FAQ.md (hunk 1 above)
Comment (typ1-brods) [reply docs/EN/Getting-Started/FAQ.md]: Thank you Adrian. Fixed.

File: docs/EN/Getting-Started/FAQ.md (hunk 2 above)
Comment (typ1-brods) [reply docs/EN/Getting-Started/FAQ.md]: Thank you Adrian. Fixed.

**Pull request merged**